ext/nnstreamer/meson.build

```diff
@@ -17,6 +17,7 @@ if protobuf_support_is_available
   pb_gen_src = pb_gen.process('./include/nnstreamer.proto')
 endif
 
+subdir('nnstreamer_protobuf')
```

Comment:
Let's rename ```/ext/nnstreamer/nnstreamer_protobuf``` to ```/ext/nnstreamer/extra/``` so that we can put similar files into the same directory later.

Reply:
Agree, I modified the path as you mentioned.